gax-java/gax/src/main/java/com/google/api/gax/rpc/ApiClientHeaderProvider.java

```diff
@@ -224,6 +247,10 @@ public Builder setApiVersionToken(String apiVersionToken) {
       return this;
     }
 
+    public String getProtobufRuntimeToken() {
```

Comment:
I know other fields have a public getter, but I don't think they have to be. Since this method is only used within Gax, can we change this to package private?

Reply:
done